lib/Secrets/secretmanager.cpp

```diff
@@ -110,6 +110,25 @@ SecretManagerPrivate::getPluginInfo()
     return reply;
 }
 
+
+QDBusPendingReply<Sailfish::Secrets::Result,
+                  HealthCheckRequest::Health,
+                  HealthCheckRequest::Health>
+SecretManagerPrivate::getHealthInfo()
+{
+    if (!m_interface) {
+        return QDBusPendingReply<Result>(
```

Comment:
this needs to return `QDBusPendingReply<Result, Health, Health>`.  I notice that the one above is also incorrect - can you please fix that one also?